<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DisplayVideo</class>
 <widget class="QWidget" name="DisplayVideo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1132</width>
    <height>1098</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QFrame" name="framePlayerControls">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">/*QFrame{
     background:rgb(249,247,248);
border: 0px;
}
QSlider::handle:horizontal {
  	background-color: rgb(0, 95, 135);		
    border: 0px outset #8f8f91;
	border-radius:0px;
    width: 30px;
    height: 48px;
    margin: -30px -30px;
}
*/</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0" colspan="7">
       <layout class="QGridLayout" name="gridLayout">
        <item row="0" column="0">
         <widget class="QSlider" name="horizontalSliderCurrentVideoFrame">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Current Video Frame Number</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/*QSlider {
    height: 48px;
   	background-color:rgb(249,247,248);	
}

QSlider::groove:horizontal {
    border: 1px solid rgb(0, 0, 0);
    height: 5px;
    background-color: rgb(64, 77, 83 );
}


QSlider::handle:horizontal {
  	background-color: rgb(0, 95, 135);		
    width: 30px;
    height: 48px;
   margin: -20px -0px;
}*/
</string>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>0</number>
          </property>
          <property name="tracking">
           <bool>true</bool>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksBothSides</enum>
          </property>
          <property name="tickInterval">
           <number>2</number>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcdNumberCurrentVideoFrameNumber">
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">QLCDNumber{
     color:rgb(39,51,66);
     background:rgb(255,255,255);
}</string>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="pushButtonStop">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Stop Video</string>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QPushButton{
    background-color: rgb(64, 77, 83 );
   color: rgb(255, 255, 255 );
   
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
      border: 2px solid rgb(148,182, 231 ); 
 }
*/

</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="pushButtonBackward">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Skip Backward</string>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QPushButton{
    background-color: rgb(64, 77, 83 );
   color: rgb(255, 255, 255 );
  
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
      border: 2px solid rgb(148,182, 231 ); 
 }
*/
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButtonStart">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Play Video</string>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QPushButton{
    background-color: rgb(64, 77, 83 );
   color: rgb(255, 255, 255 );
   
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
      border: 2px solid rgb(148,182, 231 ); 
 }
*/
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pushButtonForward">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>96</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Skip Foreward</string>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QPushButton{
    background-color: rgb(64, 77, 83 );
   color: rgb(255, 255, 255 );
   
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
      border: 2px solid rgb(148,182, 231 ); 
 }
*/
</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="4">
       <widget class="QPushButton" name="pushButtonGetNewVideo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Skip Foreward</string>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QPushButton{
    background-color: rgb(64, 77, 83 );
   color: rgb(255, 255, 255 );
   
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
      border: 2px solid rgb(148,182, 231 ); 
 }
*/
</string>
        </property>
        <property name="text">
         <string>Get Video</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QPushButton" name="pushButtonSaveVideo">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>120</width>
          <height>48</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>120</width>
          <height>48</height>
         </size>
        </property>
        <property name="toolTip">
         <string>Skip Foreward</string>
        </property>
        <property name="styleSheet">
         <string notr="true">/*QPushButton{
    background-color: rgb(64, 77, 83 );
   color: rgb(255, 255, 255 );
   
   font-size: 16px;
   font-weight: bold;
   font-family: Siemens Sans;
  border: none;
}
QPushButton:pressed {
      border: 2px solid rgb(148,182, 231 ); 
 }
*/
</string>
        </property>
        <property name="text">
         <string>Save Video</string>
        </property>
        <property name="iconSize">
         <size>
          <width>32</width>
          <height>32</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="1" column="6">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QSlider" name="horizontalSliderVideoFrameRate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>48</height>
           </size>
          </property>
          <property name="toolTip">
           <string>Video Frames Per Second</string>
          </property>
          <property name="styleSheet">
           <string notr="true">/*QSlider {
    height: 48px;
   	background-color:rgb(249,247,248);	
}

QSlider::groove:horizontal {
    border: 1px solid rgb(0, 0, 0);
    height: 5px;
    background-color: rgb(64, 77, 83 );
}


QSlider::handle:horizontal {
  	background-color: rgb(0, 95, 135);		
    width: 30px;
    height: 48px;
margin: -20px -0px;
}
*/</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>150</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="lcdNumberSetVideoFramesPerSecond">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLCDNumber{
     color:rgb(39,51,66);
     background:rgb(255,255,255);
}</string>
          </property>
          <property name="segmentStyle">
           <enum>QLCDNumber::Flat</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QFrame" name="frameliveImage">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <property name="lineWidth">
      <number>0</number>
     </property>
     <layout class="QGridLayout" name="gridLayout_6">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <layout class="QGridLayout" name="gridLayout_5"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
